docs/tutoriales/docker.md

```diff
@@ -0,0 +1,173 @@
+---
+id: docker
+title: Docker
+sidebar_label: Docker
+---
+Docker es una herramienta que permite crear, probar y desplegar aplicaciones rápidamente. La idea detrás de Docker es crear "contenedores" que sean ligeros y portátiles para que las aplicaciones puedan ejecutarse en cualquier máquina con Docker instalado, independientemente del sistema operativo.
+
+Por ejemplo, cada nodo de la red blockchain o servicio/aplicación desarrollada se compilará en una imagen Docker que contiene todas sus dependencias y se despliega en uno de los varios proveedores de infraestructura con Docker disponible.
+
+![Estructura Docker](https://upload.wikimedia.org/wikipedia/commons/thumb/0/0a/Docker-containerized-and-vm-transparent-bg.png/800px-Docker-containerized-and-vm-transparent-bg.png)
```

Comment:
instead of pulling the image from an external source lets download it and include it in our own repo